.github/workflows/changesets-release.yml

```diff
@@ -0,0 +1,115 @@
+name: "changesets Release"
+
+on:
+  pull_request_target:
+    branches:
+      - dev
+      - master
+    paths:
+      - CHANGELOG.md
+    types:
+      - closed
+
+jobs:
+  unit-tests:
+    if: github.event.pull_request.merged == true
+    uses: ./.github/workflows/reusable-unit-tests.yml
+    secrets:
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+  api-library-tests:
+    if: github.event.pull_request.merged == true
+    uses: ./.github/workflows/reusable-api-library-tests.yml
+    secrets:
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+  # nextjs-app-setup:
+  #   uses: ./.github/workflows/reusable-nextjs-app-setup.yml
+
+  package-tests:
+    if: github.event.pull_request.merged == true
+    uses: ./.github/workflows/reusable-package-tests.yml
+
+  integration-tests-on-prem:
+    if: github.event.pull_request.merged == true
+    uses: ./.github/workflows/reusable-integration-tests-on-prem.yml
+    secrets:
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+  toolbox-tests:
+    if: github.event.pull_request.merged == true
+    uses: ./.github/workflows/reusable-toolbox-tests.yml
+
+  aura-tests:
+    if: github.event.pull_request.merged == true
+    uses: ./.github/workflows/reusable-aura-tests.yml
+    secrets: inherit
```

Comment:
On second thought, it might be better to list all four Aura secrets here to be consistent with the other reusable workflows. Wdyt?